packages/components-native/src/InputSearch/InputSearch.tsx

```diff
@@ -49,11 +49,11 @@ function SearchInputInternal(
   }: InputSearchProps,
   ref: Ref<InputTextRef>,
 ) {
-  const delayedSearch = debounce(onDebouncedChange, wait);
+  const delayedSearch = debounce(() => onDebouncedChange(value), wait);
```

Comment:
I'm trying to better understand this change. looking at the `lodash` versions we have:

`"lodash": "^4.17.21",`
VS
`"lodash": "^4.17.20",`

where there's nothing that _should_ have changed around this

https://github.com/lodash/lodash/wiki/Changelog
<img width="937" alt="image" src="https://github.com/GetJobber/atlantis/assets/11843215/e4944a79-80af-4857-bd7d-7cfb746482fd">

when I go back to how this was with `delayedSearch(value)` and no wrapping arrow function around `onDebouncedChange` my linter is happy with that, which is definitely what I'd expect since `onDebouncedChange` is already function

>while building the mobile storybook example I discovered that I needed to make the first argument to debounce a function.

what do you mean by that? like the invocation in `Mobile.stories.tsx` was complaining about props, or that was fine but the example wasn't debouncing the provided `onDebouncedChange`?

actually on that note, unless I'm missing something, I don't see us passing all of the required props to our instance of `<InputSearch>`. `onChange` and `onDebouncedChange` are supposed to be mandatory.

Reply:
to be clear when I try out these changes in an actual RN context, they do seem work the same as before - I'd just like to know if we can get away with not changing them at all!

Reply:
@ZakaryH  I agree! It seems very silly considering the documentation and changelog have nothing.

The only reason I noticed this was the storybook example in the browser.
The original code on the `"lodash": "^4.17.21"` version gives me this

<img width="370" alt="Screenshot 2023-08-25 at 4 32 55 PM" src="https://github.com/GetJobber/atlantis/assets/104797704/468f89eb-c14e-4986-967c-d35a6ba2648f">

Reply:
command clicking `debounce` leads to the same result in the `function.d.ts` yet in jobber-mobile it's fine and the code is the same

Reply:
By 'result' I mean that it shows the first argument to `debounce` is a function